mock-api/mock-api-server.ts

```diff
@@ -1,19 +1,50 @@
 import * as jsonServer from 'json-server';
 import { mockTestResultList } from '../src/mocks/mock-test-result';
-import { mockVehicleTecnicalRecordList } from '../src/mocks/mock-vehicle-technical-record.mock';
+import { mockVehicleTechnicalRecordList } from '../src/mocks/mock-vehicle-technical-record.mock';
 const server = jsonServer.create();
 const router = jsonServer.router('{}');
 const middlewares = jsonServer.defaults();
 
+const args = require('minimist')(process.argv.slice(2));
```

Comment:
Can we have this as an import and add to dev dependencies in package.json?

Reply:
From documentation https://www.npmjs.com/package/minimist
Looks like this is the desired pattern